iaso/api/apps/serializers.py

```diff
@@ -123,5 +130,63 @@ def set_forms_and_feature_flags(instance, forms, feature_flags):
         if feature_flags is not None:
             instance.feature_flags.clear()
             for f_f in feature_flags:
-                f_f_object = FeatureFlag.objects.get(code=f_f["code"])
-                instance.feature_flags.add(f_f_object)
+                f_f_object = FeatureFlag.objects.get(code=f_f["featureflag"]["code"])
+                instance.feature_flags.through.save(
+                    ProjectFeatureFlags(
+                        project=instance,
+                        featureflag=f_f_object,
+                        configuration=f_f.get("configuration", None),
+                    )
+                )
```

Comment:
You're going to like this one, less noise 

```suggestion
                instance.feature_flags.add(f_f_object, through_defaults={"configuration": f_f.get("configuration", None)})
```